pkgs/tools/misc/fluent-bit/default.nix

```
@@ -1,26 +1,33 @@
-{ lib, stdenv, fetchFromGitHub, cmake, flex, bison, systemd }:
+{ lib, stdenv, fetchFromGitHub, cmake, flex, bison, pkgconfig
+, systemd, openssl }:
 
 stdenv.mkDerivation rec {
   pname = "fluent-bit";
-  version = "1.7.9";
+  version = "1.8.2";
 
   src = fetchFromGitHub {
     owner = "fluent";
     repo = "fluent-bit";
     rev = "v${version}";
-    sha256 = "sha256-rL2IJYTMs0Yzo/oqrcI56krmVnxxxRWCebSjjbjiP/k=";
+    sha256 = "1hayis4rncfs7jh5qz7q92kcivfvqsp3i98529i1vmc8yxv694r0";
   };
 
-  nativeBuildInputs = [ cmake flex bison ];
+  nativeBuildInputs = [ cmake flex bison pkgconfig ];
 
-  buildInputs = lib.optionals stdenv.isLinux [ systemd ];
+  buildInputs = [ openssl ] ++ lib.optionals stdenv.isLinux [ systemd ];
```

```suggestion
  buildInputs = [ openssl ]
    ++ lib.optionals stdenv.isLinux [ systemd ];
```
is openssl needed on darwin?

---

I don't have access to a darwin machine to test. It should at least build. I don't see code for Apple's Secure Transport: https://github.com/fluent/fluent-bit/tree/master/src/tls

We could restrict it to Linux since nobody is probably using this on Darwin anyway. :smiley: